lib/train/transports/local.rb

```diff
@@ -146,6 +146,12 @@ def initialize(powershell_cmd = "powershell")
           raise PipeError if @pipe.nil?
         end
 
+        # @param  cmd The command to execute
+        # @return Local::ComandResult with stdout, stderr and exitstatus
+        #         Note that exitstatus ($?) in PowerShell is boolean, but we use a numeric exit code.
```

Comment:
Code Climate says these comment lines are too long; but the Rakefile rubocop task doesn't mind them.
I'd rather have them this length for human readability. Do I need to shorten them?

Reply:
![image](https://user-images.githubusercontent.com/49856/68966400-d649c580-0792-11ea-978e-06b08efe1523.png)

Codeclimate is not _required_ (yet).